<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Backup</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="10,200,0,0">
        <item>
         <layout class="QGridLayout" name="gridLayout" rowstretch="5,0,0" columnstretch="15,0,0">
          <item row="2" column="0">
           <widget class="QLineEdit" name="dir_display"/>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Backup Directory</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="OpenDir">
            <property name="text">
             <string>Raw Directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="8,0,0,0,0">
          <item row="0" column="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="backupButton">
            <property name="text">
             <string>Backup</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Add File</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Overwrite</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="tab1_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Recovery</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,200,0,0">
        <item>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="5,0,0" columnstretch="15,0,0">
          <item row="2" column="2">
           <widget class="QPushButton" name="OpenDir_2">
            <property name="text">
             <string>Recovery Directory</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="dir_display_2"/>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Backup Directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="treeView_2"/>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="2,0,0,0,0">
          <item row="0" column="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="backupButton_2">
            <property name="text">
             <string>Recovery</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>U</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Overwrite</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="tab1_label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
